Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -101,6 +101,23 @@ TEST_CASE(test_jpeg_sof0_one_scan)
     EXPECT(frame.duration == 0);
 }
 
+TEST_CASE(test_jpeg_sof0_several_scans)
+{
+    auto file = MUST(Core::MappedFile::map(TEST_INPUT("several_scans.jpg"sv)));
+    EXPECT(Gfx::JPEGImageDecoderPlugin::sniff(file->bytes()));
+    auto plugin_decoder = MUST(Gfx::JPEGImageDecoderPlugin::create(file->bytes()));
+    EXPECT(plugin_decoder->initialize());
+
```

Comment:
for bonus points, describe how you created the file in the commit description of this commit

Reply:
You'd better really send me those bonus points :eyes: